test/connectors/des/DesConnectorSpec.scala

```diff
@@ -33,7 +35,7 @@ class DesConnectorSpec extends ItSpec {
     futureResponse match {
       case Some(response) => {
         response.obligations.head.obligationDetails.size shouldBe 4
-        response.obligations.head.obligationDetails.last shouldBe DesData.obligationsDataOk(vrn).
+        response.obligations.head.obligationDetails.last shouldBe DesData.obligationsDataOk(vrn, LocalDate.now().toString).
```

Comment:
Let's add to `ItSpec` frozen clock instance:


```
//example from pay-frontend:

  def frozenTimeString: String = "2027-11-02T16:33:51.880"

  lazy val overridingsModule: AbstractModule = new AbstractModule {

    override def configure(): Unit = ()

    @Provides
    @Singleton
    def clock: Clock = {
      val fixedInstant = LocalDateTime.parse(frozenTimeString).toInstant(ZoneOffset.UTC)
      Clock.fixed(fixedInstant, ZoneId.systemDefault)
    }
...

```

Reply:
Done